cirq/contrib/noise_models/noise_models.py

```diff
@@ -211,54 +212,142 @@ def noisy_moment(self, moment: 'cirq.Moment',
             ]
 
 
-class PerQubitDepolarizingNoiseModel(devices.NoiseModel):
-    """DepolarizingNoiseModel which allows depolarization probabilities to be
-    specified separately for each qubit.
+class PerQubitDepolarizingWithDampedReadoutNoiseModel(devices.NoiseModel):
+    """DepolarizingWithDampedReadoutNoiseModel which allows depolarization
```

Comment:
style nit: First line of pydoc should be a single line. No need to repeat class name.  
``Noise model with T1 added to gates and damping and bitflips applied to measurement.``

Reply:
Done. The previous comment was referencing the treats-all-qubits-equally version of DepolarizingWithDampedReadoutNoiseModel, but this should be clearer.